src/sentry/notifications/notifications/digest.py

```diff
@@ -52,8 +52,12 @@ def __init__(
         target_type: ActionTargetType,
         target_identifier: int | None = None,
         fallthrough_choice: FallthroughChoiceType | None = None,
+        notification_uuid: str | None = None,
     ) -> None:
-        super().__init__(project)
+        if notification_uuid:
+            super().__init__(project, notification_uuid)
```

Comment:
@jangjodi this seems kind of janky, can't we just pass it in as a kwarg?

Reply:
good catch - we can always pass it, as it will be `None` or a `str` and it'll be regenerated in `BaseNotification` if it is `None`